### ISSUE TITLE
<title>Data  - GRSciColl </title> to match <h1>

### ISSUE THREAD

**Issue by ckotwn** [gbif/hp-grscicoll#176]
Currently, every tab looks the same while working on multiple GRSciColl entries. It would be great to have <head><title> matching <h1> in the <body>, as the majority of web pages do. Having this should improve the efficiency of working on GRSciColl. Thanks.

<!-- Uploading "Screenshot 2024-11-01 at 15.37.30.png"... ⇢

**Comment by ckotwn** [comment]
<!-- Uploading "Screenshot 2024-11-01 at 15.37.30.png"... ⇢

**Comment by MortenHofft** [comment]
Yeah, never got around to it. But it is there in the coming version
